settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "FastAPI weather temperature management"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./weather_data.db"
```

Comment:
The `DATABASE_URL` is set to use SQLite with `aiosqlite`. Ensure that this URL is appropriate for your application's database setup. If you plan to use a different database, update the URL accordingly.